### ISSUE TITLE
video recording module is not provided by now

### ISSUE THREAD

**Issue by 5bo** [wysaid/android-gpuimage-plus#355]

**Comment by wysaid** [comment]
What?

**Comment by achinverma** [comment]
@wysaid, i am looking for the same, is it possible to record video and apply filters on live recording via using this library?

**Comment by wysaid** [comment]
sure

**Comment by wysaid** [comment]
@achinverma sure

**Comment by achinverma** [comment]
@wysaid, please can you provide any demo to record video and apply filters on live recording.
THANKS

**Comment by achinverma** [comment]
@wysaid, please can you help me with the above issue?

**Comment by wysaid** [comment]
Please produce your problem in the demo, and I take a look.

**Comment by achinverma** [comment]
@wysaid , sorry I did not get your point, I mean should I send the demo which I have in my pc?
actually, I have downloaded your demo from github and after running your demo in my moto g5 plus device, I am facing this issue.

**Comment by achinverma** [comment]
@wysaid , have you got this issue in your demo?

**Comment by wysaid** [comment]
@achinverma  I've tried the latest demo, it's all right.

**Comment by leotrinh** [comment]
> @wysaid, please can you help me with the above issue?

hi did you solved the problem? can you guide me how to export the video>

**Comment by wysaid** [comment]
> > @wysaid, please can you help me with the above issue?
> 
> hi did you solved the problem? can you guide me how to export the video>

No, I don't know the problem.  Do you have a step on the demo, to see the problem ?
